sbi/neural_nets/density_estimators/base.py

```diff
@@ -0,0 +1,61 @@
+import torch
+from torch import Tensor, nn
+
+
+class DensityEstimator:
```

Comment:
general comment: we could think about making this a `torch.Distribution`

Reply:
as discussed offline, not a good idea as not all density estimators will necessarily have, e.g. `.log_prob` method.